spec/support/shared_examples_policy_machine.rb

```diff
@@ -370,6 +370,17 @@
       policy_machine.is_privilege?(@u1, @w, @o1).should be_false
     end
 
+    context 'tolerate_cycles is set to true' do
```

Comment:
The problem is that the expected behavior is works-as-normal for most adapters, hang-and-create-a-zombie in Postgres.